active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,618 @@
+}
+```
+#### Remarks
+- **UWP**: Current behavior is [here](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.window.activated)
+
+### Window.Close method
+
+Closes and destroy the window. The Closed event occurs when a Window closes. 
+```CS
+public void Close()
+```
+
+#### Remarks
+- **UWP**: UWP app typically use a single Window instance, and do not open and close additional Window instances. Additionally, you will not typically provide custom UI to enable users to close the app window.
+
+### Window.Closed event
+
+Occurs after the window is closed. The Closed event is fired when the user closes the window, or the window is closed programmatically. Also, this causes the Activated event fires. 
+
+
+#### Samples
+
+```CS
+  window.Closed += OnClosed;
+```	
+```CS
+void OnClosed(object sender, MUX.WindowEventArgs e)
+{
+   //
+}
+```
+
+#### Remarks
+
+If this was the last window to be closed for the app:
+- **Win32**: The application ends and call Suspending event.
+- **UWP**: The application is suspended, and the Suspending event will fire. 
+
+### **Window placement management**
+
+### Window.SizeChanged event
+
+This event occurs whenever there are changes in the size of a window (e.g.: user resizes the window, user moves the window to a new display that has a different resolution and the window expands to fill it)
+
+```CS
+public event WindowSizeChangedEventHandler SizeChanged
+```
+#### Samples
+
+```CS  
+window.SizeChanged += OnSizeChanged;
+```	
+
+```CS
+void OnSizeChanged(object sender, MUX.WindowSizeChangedEventArgs e)
+{
+    if(!e.Size.IsEmpty && e.Size.Height != e.Size.Width)
+    {
+       //This is not a square Window
+    }
+}
+```
+#### Remarks
+- **UWP**: More info [here](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.window.sizechanged).
+
+### **Get the underneath window implementation**
+UWP apps use CoreWindow as a window implementation and Win32 apps use HWnds instead. The Microsoft.UI.Xaml.Window hides these detail implementations from the application developers and exposes a set of APIs that allow to create and manipulate Windows.
+These properties expose these underneath objects to enable advanced (or basic but not supported yet) scenarios, so developers can get the CoreWindow or HWnd objects to apply setters, listening events, etc.
+
+### Window.CoreWindow property: 
+
+Get the CoreWindow of a Window when the Window was created in a UWP app model.
+
+```CS
+public CoreWindow CoreWindow { get; }
+```
+
+#### Remarks
+- **Win32**: This property returns null when running in Win32 app model. 
+
+### IWindowInterop Interface 
+Exposes a COM interface (only callable from Win32) to get the HWnd of a Window in a Win32 app model.
+
+```CS
+Window w = new Window();
+w.QueryInterface(IWindowInterop);
+IntPtr hwnd = w.GetInterop<IWindowInterop>().Handle;
+```
+
+### **Others**
+### Window.Dispatcher property
+Gets the Dispatcher object for the Window.
+
+```CS
+public CoreDispatcher Dispatcher { get; }
+```
```

Comment:
Change to the DispatcherQueue, since CoreDispatcher isn't that interesting.

Reply:
It's an existing property. But DispatcherQueue is being added